src/legacy/server/saved_objects/service/lib/repository.js

```diff
@@ -57,8 +57,6 @@ export class SavedObjectsRepository {
     }
     this._allowedTypes = allowedTypes;
```

Comment:
What does allowedTypes do? Should that and _isTypeAllowed be removed completely?